pkg/allocation/converters/converter.go

```diff
@@ -303,12 +304,12 @@ func convertGameServerSelectorsToInternalGameServerSelectors(in []*pb.GameServer
 }
 
 // ConvertGSAToAllocationResponse converts GameServerAllocation V1 (GSA) to AllocationResponse
-func ConvertGSAToAllocationResponse(in *allocationv1.GameServerAllocation) (*pb.AllocationResponse, error) {
+func ConvertGSAToAllocationResponse(in *allocationv1.GameServerAllocation, grpcStatusCode codes.Code) (*pb.AllocationResponse, error) {
```

Comment:
It's lengthy but I prefer `grpcUnallocatedStatusCode` here as a name so it's clear what this code is for.

Reply:
I've renamed it to grpcUnallocatedStatusCode